src/containers/Login/index.tsx

```diff
@@ -136,86 +150,152 @@ const Login = ({ toggleSnackbar, setSnackbarMessage, setLoading }: ILoginProps)
 
   return (
     <>
-      <img src={polygon1} alt='' style={{ position: 'absolute', top: '3%', left: '-10%' }} />
```

Comment:
please don't change at background Amr did, so when you want to see your result without any conflict, commit that images or take Amr code from his PR and add it to see only the result then remove it before merging, sorry for that but it will solve after Amr background PR merged